### PR TITLE
Add test for nested schema

### PR DESCRIPTION
**Pull request by yebrahim**
Following up from https://github.com/googledatalab/pydatalab/pull/430.

### CODE REVIEW

File: tests/kernel/bigquery_tests.py [hunk 1]
```diff
@@ -46,6 +46,35 @@ def noop_decorator(func):
...
+                {'name': 'nestedField2', 'type': 'STRING'}
+              ]}
+           ]}
+        ]
+      }
+    ]
+
+    for s in schemas:
+      record = google.datalab.utils.commands.parse_config(json.dumps(s), {})
+      jsonschema.validate(record, google.datalab.bigquery.commands._bigquery.table_schema_schema)
```
Comment (jimmc) [inline tests/kernel/bigquery_tests.py]: I don't understand what this test is supposed to be testing. It seems like this test should include some good schemas and some bad schemas, but I can't tell from the code whether that is the intent.

Comment (yebrahim) [inline tests/kernel/bigquery_tests.py]: This makes sure some good scenarios are valid. Schemas with nested fields were broken until #430, so this is a regression test.

Comment (yebrahim) [inline tests/kernel/bigquery_tests.py]: Added some bad schemas for basic scenarios.

Comment (jimmc) [inline tests/kernel/bigquery_tests.py]: Thanks, that helps. It is counter-intuitive that a schema that has two 'col1' columns with different types is valid.

Comment (yebrahim) [inline tests/kernel/bigquery_tests.py]: Yes this is just a structural integrity test. Bigquery will reject the schema at the service, but we don't want to replicate all their tests on the client.

**Pull request merged**